### PR TITLE
autotest all predict types. For classif: test binary and multiclass.

### PR DESCRIPTION
**Pull request by QuayAu**
see https://github.com/mlr-org/mlr3learners/issues/13

### CODE REVIEW

**Comment by QuayAu** [comment]
I added some sanity checks: 

classif: very easy 1d separable two class task. tests if mmce < 0.3
regr: constant 1d task: f(x) = 1 for all x. tests if mse < 0.3

File: tests/testthat/test_mlr_learners_regr_featureless.R [hunk 1]
```diff
@@ -1,34 +1,7 @@
...
-test_that("Predict with se", {
-  task = mlr_tasks$get("bh")
-  learner = mlr_learners$get("regr.featureless")
-  learner$predict_type = "se"
-  expect_learner(learner, task)
-
-  e = Experiment$new(task, learner)$train()$predict()
-  expect_numeric(e$data$prediction$se, len = task$nrow, any.missing = FALSE, lower = 0)
-})
-
```
Comment (mllg) [inline tests/testthat/test_mlr_learners_regr_featureless.R]: Why have you removed these tests? Can you add them again?

Comment (QuayAu) [reply tests/testthat/test_mlr_learners_regr_featureless.R]: I thought we wanted to streamline the unit tests with an auto test for each learner. Everythin, what is tested here (simple train predict + predict type "se") is still tested with expect_autotest().

Comment (mllg) [reply tests/testthat/test_mlr_learners_regr_featureless.R]: Right, except these line:
```r
expect_class(e$model, "featureless")
expect_numeric(e$model, len = 2L, any.missing = FALSE)
```
But we can live without these tests.

**Pull request merged**